modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1786 @@
+[source, cypher]
+----
+CREATE CONSTRAINT knows_since_how
+FOR ()-[knows:KNOWS]-() REQUIRE (knows.since, knows.how) IS RELATIONSHIP KEY
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[create-key-constraint-compliant-data]]
+==== Create data that complies with existing key constraints
+
+.Create a node that complies with existing node key constraints
+======
+
+.Create an `Actor` node with `firstname` and `surname` properties that are not already in the database
+[source, cypher]
+----
+CREATE (actor:Actor {firstname: 'Keanu', surname: 'Reeves'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 label, created 1 node, set 2 properties.
+----
+
+======
+
+
+.Create a relationship that complies with existing relationship key constraints
+======
+
+.Create a `KNOWS` relationship with `since` and `how` properties that are not already in the database
+[source, cypher]
+----
+CREATE (:Actor {firstname: 'Jensen', surname: 'Ackles'})-[:KNOWS {since: 2008, how: 'coworkers', friend: true}]->(:Actor {firstname: 'Misha', surname: 'Collins'})
+----
+
+.Result
+[source, queryresult]
+----
+Added 2 labels, created 2 nodes, set 7 properties, created 1 relationship.
+----
+
+======
+
+
+[role=label--new-5.16]
+[[create-constraint-with-parameter]]
+=== Create a constraint with a parameter
+
+All constraint types can be created with a parameterized name.
+
+.Create a node property uniqueness constraint using a parameter
+======
+
+.Parameters
+[source, parameters]
+----
+{
+  "name": "node_uniqueness_param"
+}
+----
+
+.Create a node property uniqueness constraint with a parameterized name
+[source, cypher]
+----
+CREATE CONSTRAINT $name
+FOR (book:Book) REQUIRE book.prop1 IS UNIQUE
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+.Create a relationship property existence constraint using a parameter
+======
+
+.Parameters
+[source, parameters]
+----
+{
+  "name": "rel_exist_param"
+}
+----
+
+.Create a relationship property existence constraint with a parameterized name
+[source, cypher]
+----
+CREATE CONSTRAINT $name
+FOR ()-[wrote:WROTE]-() REQUIRE wrote.published IS NOT NULL
+----
+
+.Result
+[source, queryresult]
+----
+Added 1 constraint.
+----
+
+======
+
+[[multiple-constraints-on-same-property-combinations]]
+=== Multiple constraints on the same property combinations
+
+Some constraint types are allowed on the same label/relationship type and property combination.
+For example, it is possible to have a uniqueness and an existence constraint on the same label/relationship type and property combination, though this would be the equivalent of having a node or relationship key constraint.
+A more useful example would be to combine a property type and an existence constraint to ensure that the property exists and has the given type.
+
+[[constraints-and-indexes]]
+=== Constraints and backing indexes
```

Comment:
would it make sense to also have the `Constraint failures and indexes` section under the index section? 🤔  to have all 'index related' things together?